src/System.Management.Automation/engine/PSConfiguration.cs

```diff
@@ -677,5 +493,129 @@ private void RemoveValueFromFile<T>(string fileName, string key)
             }
         }
     }
-} // Namespace System.Management.Automation
 
+    #region GroupPolicy Configs
+
+    /// <summary>
+    /// The GroupPolicy related settings used in PowerShell are as follows in Registry:
+    ///  - Software\Policies\Microsoft\Windows\PowerShell -- { EnableScripts (0 or 1); ExecutionPolicy (string) }
```

Comment:
Needs to be updated as we're putting it under `...\Microsoft\PowerShellCore` now

Reply:
Will do.